x-pack/plugins/fleet/server/saved_objects/migrations/to_v7_16_0.ts

```diff
@@ -18,7 +18,7 @@ export const migrateInstallationToV7160: SavedObjectMigrationFn<Installation, In
 ) => {
   const updatedInstallationDoc = installationDoc;
 
-  if (DEFAULT_PACKAGES.some((pkg) => pkg.name === updatedInstallationDoc.attributes.name)) {
+  if (AUTO_UPDATE_PACKAGES.some((pkg) => pkg.name === updatedInstallationDoc.attributes.name)) {
```

Comment:
Given my comment above about not adding additional packages to AUTO_UPDATE_PACKAGES, we probably shouldn't update this migration from 7.16.0 or it will be slightly different behavior for users who upgrade from pre-7.16.0 to 8.1+ instead of going through 7.16.0 first. Instead, let's just hardcode this list again here for just the default packages from 7.16.0

Reply:
done